app/views/memberships/_table.html.erb

```diff
@@ -51,11 +51,11 @@
 
         <div class='govuk-grid-column-one-third govuk-list govuk-!-padding-right-4 govuk-!-margin-top-0 text-right'>
           <% if current_user.can_manage_team?(current_organisation) && member.id != current_user.id %>
-            <%= link_to "Edit permissions", edit_membership_path(member.membership_for(current_organisation)), class: "govuk-link govuk-link--no-visited-state" %>
+            <%= link_to "Edit permissions", edit_membership_path(member.membership_for(current_organisation)), class: "govuk-single-link govuk-link--no-visited-state" %>
             <% if member.totp_enabled? %>
               <%= link_to "Reset 2FA",
                           { controller: "users/two_factor_authentication_setup", action: "edit", id: member },
-                          class: "govuk-link govuk-link--no-visited-state" %>
+                          class: "govuk-single-link govuk-link--no-visited-state" %>
```

Comment:
Not using govuk-link will break goodies such as accessibility improvements we get from the library. I suggest the following approach:

* revert back to `govuk-link` on line 54 and 58
* add a new class `membership-list` on line 3 (i.e. `ul` element)
* scope the relevant CSS to this class - see comments on `team_members.scss`.

Reply:
Good idea, that makes more sense.